content/en/docs/FAQ/performance.md

```diff
@@ -3,6 +3,67 @@ title: "Performance and Scalability"
+
+Only when query performance for metrics and traces is good, can Kiali respond in a reasonable way. So, it is also important to provide sufficient connectivity for the API calls to return information in a timely way.
+
+#### Manual Refresh
+
+By default, Kiali will immediately attempt to populate the page, the default being the Overview page. After the initial page is rendered, most pages will automatically refresh, based on the setting in the "Refresh Interval" dropdown. The default is every 60s. For a large mesh, even the initial page load can be slow, and it can be frustrating if you have to wait for the page to render before being able to enter desired options and/or filters, and then ask for another refresh.
+
+In the dropdown, Kiali offers the "Manual" refresh setting. If selected, Kiali will not refresh the page on a timer. Kiali also offers a "Pause" setting. "Pause" also prevents a timed refresh, but it will refresh on an option or filter change. "Manual" will only refresh on a manual click the refresh button. To ensure that even the initial page load is avoided, the default can be set in the kiali CR:`spec.kiali_feature_flags.ui_defaults.refresh_interval: manual`. With this setting it is possible to "batch" settings changes. For example, when working with the graph you could choose namespaces, update Display settings, and also change Traffic settings, all before rendering the graph.
+
+#### URL Bookmarks
+
+Because most of Kiali is bookmarkable, it can be useful to save bookmarks of pages you like. Then, use these bookmarks to immediately apply the settings you want to the Kiali page you are visiting.
+
+#### Large Graphs
+
+Working with large graphs is difficult. A graph does not have to be very large before it becomes complicated and/or dense. Here are a few suggestions.
+
+##### Tips to reduce graph size and speed up generation
+
+- Limit the namespaces selected.
+  - Each requested namespace is like it's own graph request, and then each resulting namespace graph is "stitched" together.
+  - This may not be possible, in some mesh designs even a single namespace is very populated.
+- Reduce the protocols selected
+  - Using the Traffic dropdown, only fetch TCP or HTTP, not both. Different queries are performed for the different protocols.
+  - In Ambient, you can also choose between ztunnel and waypoint telemetry. This can reduce the number of queries and/or the size of your graph.
+- Prefer smaller Duration dropdown values.
+  - The larger the duration the more metric data being processed.
```

Comment:
```suggestion
  - The larger the duration the more metric data is being processed.
```

Reply:
I think it is OK as is,  or update your suggestion to "data that is being".